configs/mask_rcnn/mask_rcnn_vit_base_fpn_coco.py

```diff
@@ -0,0 +1,121 @@
+_base_ = [
```

Comment:
You can describe the connection with `strong_baselines` in the configuration file. I see that most of the configuration is from `strong_baselines`.

Reply:
The configuration file name should also refer to the modification in `strong_baselines`

Reply:
What is `strong_baselines`

Reply:
https://github.com/open-mmlab/mmdetection/blob/master/configs/strong_baselines/README.md